dotcom-rendering/src/lib/contributions.test.ts

```diff
@@ -176,30 +103,30 @@ describe('withinLocalNoBannerCachePeriod', () => {
 	});
 });
 
-describe('hasSupporterCookie', () => {
+describe('shouldHideSupportMessaging', () => {
```

Comment:
`shouldHideSupportMessging` has the same input and output as `hasSupporterCookie` once the one-time stuff is removed, so `hasSupporterCookie has been removed.